readme.md

```diff
@@ -451,9 +451,8 @@ was provided.
 1. Clone the repo: `git clone https://github.com/citycide/param.macro.git`
 2. Move into the new directory: `cd param.macro`
 3. Install dependencies: `npm install`
-4. Link the project to itself: `npm link && npm link param.macro`
-5. Build the source: `npm run build`
-6. Run tests: `npm test`
+4. Build the source: `npm run build`
+5. Run tests: `npm test`
 
 > this project uses itself in its source, so take note of the `npm link`
```

Comment:
This note about `npm link` needs to be updated since normally it'll be done automatically. How about we link to the readme at `npm-self-link` where we describe what "bootstrapping" is, how it allows us to use a package in itself, and a small blurb about `npm link`?

> this project uses itself in its source &mdash; see the [`npm-self-link`](https://github.com/Andarist/npm-self-link) readme for more info on bootstrapping

Reply:
Made a PR to `npm-self-link` to document it. Once that's in we can update this note and should be good to go.

Reply:
I've merged the other PR - thanks! Will you be able to finish this PR here? I've wanted to do it myself before but don't have much time lately :s

Reply:
Yeah I'll finish it up, quick fix from here anyway.

Reply:
I hit issues with `npm-self-link` on Windows, still investigating. We'll probably have to use something like `cross-spawn` or `execa`, I believe it's a known issue with node's `spawn`.